pkg/occlient/fakeclient.go

```diff
@@ -0,0 +1,24 @@
+package occlient
+
+import (
+	fkrouteClientset "github.com/openshift/client-go/route/clientset/versioned/fake"
```

Comment:
would fakeRouteClientSet be a better name?

Reply:
you are right. I've changed it.